From 588bd914f920c3f52d7d65b50c64bd4ff7906626 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Tue, 4 Sep 2018 01:41:10 +0200
Subject: [PATCH 13/13] stagefright: Move QCOM_BSP_LEGACY flag to correct
 blueprint file

In Pie the bufferqueue sources were moved to a new folder.

Change-Id: If56dd48a178658b8b8e261ef0d3851f7fbdc8475
---
 media/libstagefright/bqhelper/Android.bp | 8 ++++++++
 media/libstagefright/omx/Android.bp      | 8 --------
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/media/libstagefright/bqhelper/Android.bp b/media/libstagefright/bqhelper/Android.bp
index 4f46be788..7cadbef6e 100644
--- a/media/libstagefright/bqhelper/Android.bp
+++ b/media/libstagefright/bqhelper/Android.bp
@@ -55,6 +55,14 @@ cc_library_shared {
         "-Wno-documentation",
     ],
 
+    product_variables: {
+        lineage: {
+            uses_qcom_bsp_legacy: {
+                cppflags: ["-DQCOM_BSP_LEGACY"],
+            },
+        },
+    },
+
     sanitize: {
         misc_undefined: [
             "signed-integer-overflow",
diff --git a/media/libstagefright/omx/Android.bp b/media/libstagefright/omx/Android.bp
index 139f37919..3e6942bc8 100644
--- a/media/libstagefright/omx/Android.bp
+++ b/media/libstagefright/omx/Android.bp
@@ -72,14 +72,6 @@ cc_library_shared {
         "-Wno-documentation",
     ],
 
-    product_variables: {
-        lineage: {
-            uses_qcom_bsp_legacy: {
-                cppflags: ["-DQCOM_BSP_LEGACY"],
-            },
-        },
-    },
-
     sanitize: {
         misc_undefined: [
             "signed-integer-overflow",
-- 
2.17.1

